### DISCUSSION TITLE
App.을 실제기기에서 구동하는 방법은 책에 어디에 있을까요?

### DISCUSSION THREAD

**Discussion by JeesubKim** [velopert/dealing-with-react-native#6] [Q&A] [answered]
안녕하세요~ 리액트네이티브를 다루는 기술을 구매하여 잘 보고있습니다. 좋은책 감사합니다.

다름이 아니오라 
`1.3.1 안드로이드 환경에서 리액트 네이티브 앱 구동하기` 에 
```
이번장에서는 시뮬레이터로 구동하는 방법을 알아보고, 실제 기기에서 구동하는 방법은 나중에 알아보겠습니다. 
```

라고 되어있어서 찾아보려고 했는데 실제 기기에서 구동하는방법이 책 어디에 나와있을까요? ㅠ 

혹시 없으면 참고할만한 링크를 가이드 해주실 수 있을까요?

**Comment by velopert** [comment, marked as answer]
늦게 회신드려 죄송합니다.
실기기에서 실행하는게 iOS 내용만 포함되어있었군요..
Android 실기기에서 실행하는 내용을 다음 중쇄때 추가해놓겠습니다.


나중에 보실 분들을 위해 적어놓자면 - 
다음 링크를 참조하여 Android 개발자 모드를 활성화하고,

https://developer.android.com/studio/debug/dev-options?hl=ko
yarn android 를 입력하면 구동이 됩니다.

릴리즈 빌드의 겨웅엔 16.2.3 를 통해 구동할 수 있습니다.
